src/Cassandra/Data/Linq/CqlExpressionVisitor.cs

```diff
@@ -1118,19 +1122,32 @@ private static object GetClosureValue(MemberExpression node)
 
         private static object GetFieldValue(MemberExpression node)
         {
-            var fieldInfo = (FieldInfo)node.Member;
+            MemberOrPropertyGetValue getValueDel;
```

Comment:
If you want to avoid creating the delegate, you can reuse the generic one: `Func<object, object>`